src/classes/Logger.ts

```diff
@@ -0,0 +1,29 @@
+import * as winston from 'winston';
+import ConfigurationManager from './ConfigurationManager';
+import * as moment from 'moment';
+
+class Logger {
+
+    private loggerInstance: winston.Logger;
+
+    constructor (private configurationManager: ConfigurationManager) {
```

Comment:
Since configurationManager is used once in constructor, I think you can remove the `private`

Reply:
Nice Logger implementation